addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -52,21 +55,27 @@ var RunningTourActionHelper = core.Class.extend({
     },
     _click: function (values) {
         var href = values.$element.attr("href");
-        if (href && href.length && href[0] !== "#" && values.$element.is("a")) {
+        if (href && href.length && href[0] !== "#" && values.$element.is("a") && href !== 'javascript:void(0)') {
             window.location.href = href;
         } else {
-            values.$element.mousedown().mouseup().click();
+            values.$element.trigger("mouseenter").mousedown().mouseup().click().trigger("mouseleave");
         }
     },
     _text: function (values, text) {
         this._click(values);
 
         text = text || "Test";
-        if (values.consume_event === "input") {
-            values.$element.val(text).trigger("input");
+        if (values.$element.is('select')) {
+            values.$element.find('option').removeProp('selected')
+                .filter(function () { return $(this).val() == text; })
+                .prop('selected', 'selected');
+            values.$element.mousedown().mouseup().click();
+        } else if (values.consume_event === "input") {
+            values.$element.trigger('keydown').val(text).trigger('keyup').trigger("input");
         } else {
             values.$element.text(text);
         }
+        values.$element.trigger("change");
```

Comment:
Yes, need for select, input (because trigger input but not change in automatic mode) and for content change (when edit content with editor)